packages/create-guten-block/app/createGitignore.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Install a starter .gitignore
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const fs = require( 'fs-extra' );
+const shell = require( 'shelljs' );
+
+module.exports = (blockDir) => {
+	shell.cd( blockDir );
+	shell.touch( '.gitignore' );
+
+	// Build a default .gitignore
+	const ignore = `node_modules\ndist/`;
```

Comment:
Why are you adding `dist/` in it? That should be a part of your repo.

Reply:
I guess it is a matter of personal preference: most CLI tools like this (`create-react-app` and `vue-cli`, for example) will generate projects that exclude compiled files, and I personally keep anything that's compiled out of VC for sanity when merging, etc.

I left it in but as commented out (with a little disclaimer) so that users that know what they're doing can uncomment it.